zeppelin-interpreter/pom.xml

```diff
@@ -36,6 +36,14 @@
   <url>http://zeppelin.incubator.apache.org</url>
 
   <dependencies>
+
+    <dependency>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>zeppelin-spark-utils</artifactId>
```

Comment:
Zeppelin-interpreter is common dependency for all Interpreter implementations and need to minimize it's dependencies.  Isn't it better 'spark' has 'zeppelin-spark-utils' as a dependency?